includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+
+class DesignSystemCommunityHeaderModel extends WikiaModel {
+	const WORDMARK_TYPE_GRAPHIC = 'graphic';
+
+	private $product;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$product" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)